velox/expression/FunctionSignature.cpp

```diff
@@ -145,8 +145,8 @@ void validate(
     const std::unordered_map<std::string, SignatureVariable>& variables,
     const TypeSignature& returnType,
     const std::vector<TypeSignature>& argumentTypes,
-    const std::vector<bool>& constantArguments) {
-  std::unordered_set<std::string> usedVariables;
+    const std::vector<bool>& constantArguments,
+    std::unordered_set<std::string> usedVariables) {
```

Comment:
Is this an input parameter? Pass by const &

Reply:
`validateBaseTypeAndCollectTypeParams` will add elements to this set, so it cannot be const &.